Source: Gentoo (08_use_shared_expat.patch)
Upstream: ?
Reason: Use the shared system expat library.

--- boost_1_35_0.orig/libs/graph/build/Jamfile.v2	2008-04-24 16:00:34.000000000 +0200
+++ boost_1_35_0/libs/graph/build/Jamfile.v2	2008-06-05 14:07:16.000000000 +0200
@@ -26,7 +26,7 @@
   optional_sources += graphml.cpp ;
   optional_reqs += <include>$(EXPAT_INCLUDE) 
                    <library-path>$(EXPAT_LIBPATH) 
-                   <find-static-library>expat
+                   <find-shared-library>expat
                 ;
 }
 else
